src/include/parser/expression/parameter_value_expression.h

```diff
@@ -57,6 +65,9 @@ class ParameterValueExpression : public AbstractExpression {
 
  private:
   // TODO(Tianyu): Can we get a better name for this?
+  /**
+   * Offset of the parameter
```

Comment:
"Offset of the value that this expression points to in the query's parameter list"